src/main/java/com/northconcepts/datapipeline/examples/cookbook/ReadUuidFromPostgreSQL.java

```diff
@@ -0,0 +1,60 @@
+package com.northconcepts.datapipeline.examples.cookbook;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+
+import com.northconcepts.datapipeline.core.DataWriter;
+import com.northconcepts.datapipeline.core.StreamWriter;
+import com.northconcepts.datapipeline.internal.test.DataPipelineUnitTest;
+import com.northconcepts.datapipeline.jdbc.JdbcReader;
+import com.northconcepts.datapipeline.job.Job;
+
+public class ReadUuidFromPostgreSQL extends DataPipelineUnitTest {
+
+    final static String DATABASE_DRIVER = "org.postgresql.Driver";
+    final static String DATABASE_URL = "jdbc:postgresql://localhost:52200/test";
```

Comment:
1. Let's use the default Postgres port to prevent any confusion.  
2. Let's also name the database something other than "test" (maybe "examples" or similar).
3. Do we need to add the `&stringtype=unspecified` query param for the UUIDs to be inserted from strings?